internal/kafka/internal/services/kafka.go

```diff
@@ -1036,6 +1037,9 @@ func buildManagedKafkaCR(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 		"bf2.org/kafkaInstanceProfileQuotaConsumed": strconv.Itoa(k.QuotaConsumed),
 		"bf2.org/kafkaInstanceProfileType":          kafkaRequest.InstanceType,
 	}
+	if strings.HasPrefix(kafkaRequest.Status, "suspend") {
+		labels["bf2.org/suspended"] = fmt.Sprintf("%t", true)
+	}
```

Comment:
We could use https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/utilities/arrays.md#contains over here: 

```suggestion 
                suspendedStatuses := []string{KafkaRequestStatusSuspending.String(), KafkaRequestStatusSuspended.String()}
		labels["bf2.org/suspended"] = fmt.Sprintf("%t", arrays.Contains(suspendedStatuses, kafkaRequest.Status))
```

Reply:
Worth noting that the suggestion will add the label all the time: either `true`  or `false`. For existing Kafkas this will cause some pods restart but it won't cause service disruption.

Reply:
makes sense to use the array, however I think the label should only be sent if suspended is true (according to the epic brief). not sure if we want to send `suspended=false`

Reply:
Sending `false` should be fine as well and it'll serve us that one extra `if`: Please see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/blob/4b570dec3c121012e1a330fc4f2d0388c1200d67/api/src/main/java/org/bf2/operator/resources/v1alpha1/ManagedKafka.java#L203 which is part of https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/pull/771

@MikeEdgar is my understanding above correct?

Reply:
@machi1990, yes that is correct. `false` is the same as the label being absent.